parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/PLSQL.g4

```diff
@@ -574,15 +588,15 @@ plsqlTriggerSource
     ;
 
 simpleDmlTrigger
-    : (BEFORE | AFTER) dmlEventClause (FOR EACH ROW)? triggerBody
+    : (BEFORE | AFTER) dmlEventClause (FOR EACH ROW)? () ? triggerBody
```

Comment:
Is this writing legal in Oracle?

Reply:
Okay, I'll make the changes right away.

Reply:
I have finished the modifications now.

Reply:
> I have finished the modifications now.

@imzhangds Good job. Do you want to get more deeply involved in the ShardingSphere? If you are interested, please contact me.